cmd/cli/kubectl-kyverno/apply/apply_command.go

```diff
@@ -156,7 +157,7 @@ func Command() *cobra.Command {
 				return err
 			}
 
-			PrintReportOrViolation(applyCommandConfig.PolicyReport, rc, applyCommandConfig.ResourcePaths, len(resources), skipInvalidPolicies, applyCommandConfig.Stdin, pvInfos)
+			PrintReportOrViolation(applyCommandConfig.PolicyReport, rc, applyCommandConfig.ResourcePaths, len(resources), skipInvalidPolicies, applyCommandConfig.Stdin, pvInfos, applyCommandConfig.WarnExitCode)
```

Comment:
Why do we need to pass an additional argument when the var is already accessible in the package ?

Reply:
From  the `PrintReportOrViolation` func I don't see how/where this would be available without passing in the value to this func. The same as passing in `applyCommandConfig.PolicyReport`, etc here.. But, go isn't my primary language, so I may be missing something silly here.. 🤔 

I could init a global var and assign the value before we call `PrintReportOrViolation()` but that doesn't blow my skirt up too much.

Reply:
Ah yes, it's not global, my mistake.